ovn-tester/ovn_utils.py

```diff
@@ -610,6 +610,9 @@ def create_lb(self, name, protocol):
             protocol=protocol,
             get_func=partial(self.idl.lb_get, lb_name),
         )
+        self.idl.db_set(
+            "Load_Balancer", uuid, ("options", {"neighbor_responder": "none"})
+        ).execute()
```

Comment:
A bit of a nit pick but would it be possible to set the option in the same transaction as the LB creation?

Reply:
TBH, I got a bit lost in python trying to figure out how to pass parameters into `self.db_create_transaction`.  I'll try one more time.

Reply:
OK.  Updated version seems to work fine.